### ISSUE TITLE
Datepicker accessibility issues

### ISSUE THREAD

**Issue by vibhava** [OfficeDev/office-ui-fabric-react#1303]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) Unsure

Requested priority: (Blocking, High, Normal, Low)  High

#### Describe the issue:

On tabbing to the datepicker and opening, I can use arrows to move to different dates in a month but do not get screen reader readout . I also cannot access the arrows to change the month with keyboard. 

#### Actual behavior:

On tabbing to the datepicker and opening, I can use arrows to move to different dates in a month but do not get screen reader readout . I also cannot access the arrows to change the month with keyboard. 

#### Expected behavior:

The Fabric datepicker should be easily accessible with keyboard, and with screen reader. 
If there is a way to access with keyboard and get readout with a screen reader that I am missing, could you please let me know? Thank you.

**Comment by MLoughry** [comment]
I will have a PR shortly to address the readout issue.

Regarding the keyboard navigation, it is currently working as intended. You can navigate to the month/year arrows using (shift+)tab, and you can even navigate to the previous/next month by using the arrow keys to navigate beyond the edges of the currently selected month.

**Comment by vibhava** [comment]
Hi Michael,

I apologise for the late response, I was on vacation. Yes, I am able to see
what you mention now, and am able to navigate to the previous/next months
using the tab/shift tab keys.

I will look at the changes in the PR. Thank you for your response. I was
also seeing an issue with the date dropdown staying open on tabbing away
earlier, but not seeing that anymore.

Thank you for taking a look at the issue.

Regards,
Vibha
